src/Menu.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'item'
+#require 'twilio-ruby'
+
+class Menu
+    attr_reader :basket, :menu
+
+    def initialize()
+        @basket = []
+        @menu = [Item.new("Pizza", 3.99), Item.new("Burger", 4.99), Item.new("Fries", 2.99)]
```

Comment:
We could break this out into two objects - one responsible for displaying the menu and understanding what is available, and another responsible for keeping track of the things that are ordered.